templates/repo/view_file.tmpl

```diff
@@ -137,4 +140,8 @@ function submitDeleteForm() {
 		$("#delete-file-form").submit()
 	}
 }
+function copyLinkHref(link) {
```

Comment:
this should be moved to index.js

Reply:
I can accept the `copyLinkHref` appears here, since we already have legacy scripts above.

And `index.js` should never be added any more code in my mind, it is too big, even more, `copyLinkHref` is too simple to be put into a separated js file.

So I think we can just approve and refactor all the legacy code together after https://github.com/go-gitea/gitea/issues/17149 (the frontend refactor)

Reply:
Should I change it ore not?

Reply:
Maybe you can look into `features/clipboard.js` and try use the mechanism provided by framework: `[data-clipboard-text]`. Anyway re-using stable code is a good choice. 😊

Reply:
> Maybe you can look into features/clipboard.js and try to use the mechanism provided by framework: [data-clipboard-text]. Anyway re-using stable code is a good choice. :blush: 

I changed it to this here: https://github.com/go-gitea/gitea/pull/17145/commits/3adc23d8f7471446f83abc81ee63f57068efb7c1